syntax = "proto3";

package protobuf;

message TestMsg20 {
    string field1 =1;
    string field2 =2;
    string field3 =3;
    string field4 =4;
    string field5 =5;
    string field6 =6;
    string field7 =7;
    string field8 =8;
    string field9 =9;
    string field10 =10;
    string field11 =11;
    string field12 =12;
    string field13 =13;
    string field14 =14;
    string field15 =15;
    string field16 =16;
    string field17 =17;
    string field18 =18;
    string field19 =19;
    string field20 =20;
}

message TestMsg100 {
    int32 field1 = 1;
    int32 field2 = 2;
    int32 field3 = 3;
    int32 field4 = 4;
    int32 field5 = 5;
    int32 field6 = 6;
    int32 field7 = 7;
    int32 field8 = 8;
    int32 field9 = 9;
    int32 field10 = 10;
    int32 field11 = 11;
    int32 field12 = 12;
    int32 field13 = 13;
    int32 field14 = 14;
    int32 field15 = 15;
    int32 field16 = 16;
    int32 field17 = 17;
    int32 field18 = 18;
    int32 field19 = 19;
    int32 field20 = 20;
    int32 field21 = 21;
    int32 field22 = 22;
    int32 field23 = 23;
    int32 field24 = 24;
    int32 field25 = 25;
    int32 field26 = 26;
    int32 field27 = 27;
    int32 field28 = 28;
    int32 field29 = 29;
    int32 field30 = 30;
    int32 field31 = 31;
    int32 field32 = 32;
    int32 field33 = 33;
    int32 field34 = 34;
    int32 field35 = 35;
    int32 field36 = 36;
    int32 field37 = 37;
    int32 field38 = 38;
    int32 field39 = 39;
    int32 field40 = 40;
    float field41 = 41;
    float field42 = 42;
    float field43 = 43;
    float field44 = 44;
    float field45 = 45;
    float field46 = 46;
    float field47 = 47;
    float field48 = 48;
    float field49 = 49;
    float field50 = 50;
    float field51 = 51;
    float field52 = 52;
    float field53 = 53;
    float field54 = 54;
    float field55 = 55;
    float field56 = 56;
    float field57 = 57;
    float field58 = 58;
    float field59 = 59;
    float field60 = 60;
    float field61 = 61;
    float field62 = 62;
    float field63 = 63;
    float field64 = 64;
    float field65 = 65;
    float field66 = 66;
    float field67 = 67;
    float field68 = 68;
    float field69 = 69;
    float field70 = 70;
    string field71 = 71;
    string field72 = 72;
    string field73 = 73;
    string field74 = 74;
    string field75 = 75;
    string field76 = 76;
    string field77 = 77;
    string field78 = 78;
    string field79 = 79;
    string field80 = 80;
    string field81 = 81;
    string field82 = 82;
    string field83 = 83;
    string field84 = 84;
    string field85 = 85;
    string field86 = 86;
    string field87 = 87;
    string field88 = 88;
    string field89 = 89;
    string field90 = 90;
    string field91 = 91;
    string field92 = 92;
    string field93 = 93;
    string field94 = 94;
    string field95 = 95;
    string field96 = 96;
    string field97 = 97;
    string field98 = 98;
    string field99 = 99;
    string field100 = 100;
}

message TestMsg500 {
    int32 field1 = 1;
    int32 field2 = 2;
    int32 field3 = 3;
    int32 field4 = 4;
    int32 field5 = 5;
    int32 field6 = 6;
    int32 field7 = 7;
    int32 field8 = 8;
    int32 field9 = 9;
    int32 field10 = 10;
    int32 field11 = 11;
    int32 field12 = 12;
    int32 field13 = 13;
    int32 field14 = 14;
    int32 field15 = 15;
    int32 field16 = 16;
    int32 field17 = 17;
    int32 field18 = 18;
    int32 field19 = 19;
    int32 field20 = 20;
    int32 field21 = 21;
    int32 field22 = 22;
    int32 field23 = 23;
    int32 field24 = 24;
    int32 field25 = 25;
    int32 field26 = 26;
    int32 field27 = 27;
    int32 field28 = 28;
    int32 field29 = 29;
    int32 field30 = 30;
    int32 field31 = 31;
    int32 field32 = 32;
    int32 field33 = 33;
    int32 field34 = 34;
    int32 field35 = 35;
    int32 field36 = 36;
    int32 field37 = 37;
    int32 field38 = 38;
    int32 field39 = 39;
    int32 field40 = 40;
    int32 field41 = 41;
    int32 field42 = 42;
    int32 field43 = 43;
    int32 field44 = 44;
    int32 field45 = 45;
    int32 field46 = 46;
    int32 field47 = 47;
    int32 field48 = 48;
    int32 field49 = 49;
    int32 field50 = 50;
    int32 field51 = 51;
    int32 field52 = 52;
    int32 field53 = 53;
    int32 field54 = 54;
    int32 field55 = 55;
    int32 field56 = 56;
    int32 field57 = 57;
    int32 field58 = 58;
    int32 field59 = 59;
    int32 field60 = 60;
    int32 field61 = 61;
    int32 field62 = 62;
    int32 field63 = 63;
    int32 field64 = 64;
    int32 field65 = 65;
    int32 field66 = 66;
    int32 field67 = 67;
    int32 field68 = 68;
    int32 field69 = 69;
    int32 field70 = 70;
    int32 field71 = 71;
    int32 field72 = 72;
    int32 field73 = 73;
    int32 field74 = 74;
    int32 field75 = 75;
    int32 field76 = 76;
    int32 field77 = 77;
    int32 field78 = 78;
    int32 field79 = 79;
    int32 field80 = 80;
    int32 field81 = 81;
    int32 field82 = 82;
    int32 field83 = 83;
    int32 field84 = 84;
    int32 field85 = 85;
    int32 field86 = 86;
    int32 field87 = 87;
    int32 field88 = 88;
    int32 field89 = 89;
    int32 field90 = 90;
    int32 field91 = 91;
    int32 field92 = 92;
    int32 field93 = 93;
    int32 field94 = 94;
    int32 field95 = 95;
    int32 field96 = 96;
    int32 field97 = 97;
    int32 field98 = 98;
    int32 field99 = 99;
    int32 field100 = 100;
    int32 field101 = 101;
    int32 field102 = 102;
    int32 field103 = 103;
    int32 field104 = 104;
    int32 field105 = 105;
    int32 field106 = 106;
    int32 field107 = 107;
    int32 field108 = 108;
    int32 field109 = 109;
    int32 field110 = 110;
    int32 field111 = 111;
    int32 field112 = 112;
    int32 field113 = 113;
    int32 field114 = 114;
    int32 field115 = 115;
    int32 field116 = 116;
    int32 field117 = 117;
    int32 field118 = 118;
    int32 field119 = 119;
    int32 field120 = 120;
    int32 field121 = 121;
    int32 field122 = 122;
    int32 field123 = 123;
    int32 field124 = 124;
    int32 field125 = 125;
    int32 field126 = 126;
    int32 field127 = 127;
    int32 field128 = 128;
    int32 field129 = 129;
    int32 field130 = 130;
    int32 field131 = 131;
    int32 field132 = 132;
    int32 field133 = 133;
    int32 field134 = 134;
    int32 field135 = 135;
    int32 field136 = 136;
    int32 field137 = 137;
    int32 field138 = 138;
    int32 field139 = 139;
    int32 field140 = 140;
    int32 field141 = 141;
    int32 field142 = 142;
    int32 field143 = 143;
    int32 field144 = 144;
    int32 field145 = 145;
    int32 field146 = 146;
    int32 field147 = 147;
    int32 field148 = 148;
    int32 field149 = 149;
    int32 field150 = 150;
    int32 field151 = 151;
    int32 field152 = 152;
    int32 field153 = 153;
    int32 field154 = 154;
    int32 field155 = 155;
    int32 field156 = 156;
    int32 field157 = 157;
    int32 field158 = 158;
    int32 field159 = 159;
    int32 field160 = 160;
    int32 field161 = 161;
    int32 field162 = 162;
    int32 field163 = 163;
    int32 field164 = 164;
    int32 field165 = 165;
    int32 field166 = 166;
    int32 field167 = 167;
    int32 field168 = 168;
    int32 field169 = 169;
    int32 field170 = 170;
    int32 field171 = 171;
    int32 field172 = 172;
    int32 field173 = 173;
    int32 field174 = 174;
    int32 field175 = 175;
    int32 field176 = 176;
    int32 field177 = 177;
    int32 field178 = 178;
    int32 field179 = 179;
    int32 field180 = 180;
    int32 field181 = 181;
    int32 field182 = 182;
    int32 field183 = 183;
    int32 field184 = 184;
    int32 field185 = 185;
    int32 field186 = 186;
    int32 field187 = 187;
    int32 field188 = 188;
    int32 field189 = 189;
    int32 field190 = 190;
    int32 field191 = 191;
    int32 field192 = 192;
    int32 field193 = 193;
    int32 field194 = 194;
    int32 field195 = 195;
    int32 field196 = 196;
    int32 field197 = 197;
    int32 field198 = 198;
    int32 field199 = 199;
    int32 field200 = 200;
    float field201 = 201;
    float field202 = 202;
    float field203 = 203;
    float field204 = 204;
    float field205 = 205;
    float field206 = 206;
    float field207 = 207;
    float field208 = 208;
    float field209 = 209;
    float field210 = 210;
    float field211 = 211;
    float field212 = 212;
    float field213 = 213;
    float field214 = 214;
    float field215 = 215;
    float field216 = 216;
    float field217 = 217;
    float field218 = 218;
    float field219 = 219;
    float field220 = 220;
    float field221 = 221;
    float field222 = 222;
    float field223 = 223;
    float field224 = 224;
    float field225 = 225;
    float field226 = 226;
    float field227 = 227;
    float field228 = 228;
    float field229 = 229;
    float field230 = 230;
    float field231 = 231;
    float field232 = 232;
    float field233 = 233;
    float field234 = 234;
    float field235 = 235;
    float field236 = 236;
    float field237 = 237;
    float field238 = 238;
    float field239 = 239;
    float field240 = 240;
    float field241 = 241;
    float field242 = 242;
    float field243 = 243;
    float field244 = 244;
    float field245 = 245;
    float field246 = 246;
    float field247 = 247;
    float field248 = 248;
    float field249 = 249;
    float field250 = 250;
    float field251 = 251;
    float field252 = 252;
    float field253 = 253;
    float field254 = 254;
    float field255 = 255;
    float field256 = 256;
    float field257 = 257;
    float field258 = 258;
    float field259 = 259;
    float field260 = 260;
    float field261 = 261;
    float field262 = 262;
    float field263 = 263;
    float field264 = 264;
    float field265 = 265;
    float field266 = 266;
    float field267 = 267;
    float field268 = 268;
    float field269 = 269;
    float field270 = 270;
    float field271 = 271;
    float field272 = 272;
    float field273 = 273;
    float field274 = 274;
    float field275 = 275;
    float field276 = 276;
    float field277 = 277;
    float field278 = 278;
    float field279 = 279;
    float field280 = 280;
    float field281 = 281;
    float field282 = 282;
    float field283 = 283;
    float field284 = 284;
    float field285 = 285;
    float field286 = 286;
    float field287 = 287;
    float field288 = 288;
    float field289 = 289;
    float field290 = 290;
    float field291 = 291;
    float field292 = 292;
    float field293 = 293;
    float field294 = 294;
    float field295 = 295;
    float field296 = 296;
    float field297 = 297;
    float field298 = 298;
    float field299 = 299;
    float field300 = 300;
    float field301 = 301;
    float field302 = 302;
    float field303 = 303;
    float field304 = 304;
    float field305 = 305;
    float field306 = 306;
    float field307 = 307;
    float field308 = 308;
    float field309 = 309;
    float field310 = 310;
    float field311 = 311;
    float field312 = 312;
    float field313 = 313;
    float field314 = 314;
    float field315 = 315;
    float field316 = 316;
    float field317 = 317;
    float field318 = 318;
    float field319 = 319;
    float field320 = 320;
    float field321 = 321;
    float field322 = 322;
    float field323 = 323;
    float field324 = 324;
    float field325 = 325;
    float field326 = 326;
    float field327 = 327;
    float field328 = 328;
    float field329 = 329;
    float field330 = 330;
    float field331 = 331;
    float field332 = 332;
    float field333 = 333;
    float field334 = 334;
    float field335 = 335;
    float field336 = 336;
    float field337 = 337;
    float field338 = 338;
    float field339 = 339;
    float field340 = 340;
    float field341 = 341;
    float field342 = 342;
    float field343 = 343;
    float field344 = 344;
    float field345 = 345;
    float field346 = 346;
    float field347 = 347;
    float field348 = 348;
    float field349 = 349;
    float field350 = 350;
    float field351 = 351;
    float field352 = 352;
    float field353 = 353;
    float field354 = 354;
    float field355 = 355;
    float field356 = 356;
    float field357 = 357;
    float field358 = 358;
    float field359 = 359;
    float field360 = 360;
    float field361 = 361;
    float field362 = 362;
    float field363 = 363;
    float field364 = 364;
    float field365 = 365;
    float field366 = 366;
    float field367 = 367;
    float field368 = 368;
    float field369 = 369;
    float field370 = 370;
    float field371 = 371;
    float field372 = 372;
    float field373 = 373;
    float field374 = 374;
    float field375 = 375;
    float field376 = 376;
    float field377 = 377;
    float field378 = 378;
    float field379 = 379;
    float field380 = 380;
    float field381 = 381;
    float field382 = 382;
    float field383 = 383;
    float field384 = 384;
    float field385 = 385;
    float field386 = 386;
    float field387 = 387;
    float field388 = 388;
    float field389 = 389;
    float field390 = 390;
    float field391 = 391;
    float field392 = 392;
    float field393 = 393;
    float field394 = 394;
    float field395 = 395;
    float field396 = 396;
    float field397 = 397;
    float field398 = 398;
    float field399 = 399;
    float field400 = 400;
    string field401 = 401;
    string field402 = 402;
    string field403 = 403;
    string field404 = 404;
    string field405 = 405;
    string field406 = 406;
    string field407 = 407;
    string field408 = 408;
    string field409 = 409;
    string field410 = 410;
    string field411 = 411;
    string field412 = 412;
    string field413 = 413;
    string field414 = 414;
    string field415 = 415;
    string field416 = 416;
    string field417 = 417;
    string field418 = 418;
    string field419 = 419;
    string field420 = 420;
    string field421 = 421;
    string field422 = 422;
    string field423 = 423;
    string field424 = 424;
    string field425 = 425;
    string field426 = 426;
    string field427 = 427;
    string field428 = 428;
    string field429 = 429;
    string field430 = 430;
    string field431 = 431;
    string field432 = 432;
    string field433 = 433;
    string field434 = 434;
    string field435 = 435;
    string field436 = 436;
    string field437 = 437;
    string field438 = 438;
    string field439 = 439;
    string field440 = 440;
    string field441 = 441;
    string field442 = 442;
    string field443 = 443;
    string field444 = 444;
    string field445 = 445;
    string field446 = 446;
    string field447 = 447;
    string field448 = 448;
    string field449 = 449;
    string field450 = 450;
    string field451 = 451;
    string field452 = 452;
    string field453 = 453;
    string field454 = 454;
    string field455 = 455;
    string field456 = 456;
    string field457 = 457;
    string field458 = 458;
    string field459 = 459;
    string field460 = 460;
    string field461 = 461;
    string field462 = 462;
    string field463 = 463;
    string field464 = 464;
    string field465 = 465;
    string field466 = 466;
    string field467 = 467;
    string field468 = 468;
    string field469 = 469;
    string field470 = 470;
    string field471 = 471;
    string field472 = 472;
    string field473 = 473;
    string field474 = 474;
    string field475 = 475;
    string field476 = 476;
    string field477 = 477;
    string field478 = 478;
    string field479 = 479;
    string field480 = 480;
    string field481 = 481;
    string field482 = 482;
    string field483 = 483;
    string field484 = 484;
    string field485 = 485;
    string field486 = 486;
    string field487 = 487;
    string field488 = 488;
    string field489 = 489;
    string field490 = 490;
    string field491 = 491;
    string field492 = 492;
    string field493 = 493;
    string field494 = 494;
    string field495 = 495;
    string field496 = 496;
    string field497 = 497;
    string field498 = 498;
    string field499 = 499;
    string field500 = 500;
}
